artifactory.py

```diff
@@ -707,13 +707,15 @@ def rest_post(
         verify=True,
         cert=None,
         timeout=None,
+        json_data=None,
```

Comment:
I'm not sure that we should call it `json_data`. Json it's just a format, in python we use dict\list and so on to represent json structre.
Just `data` would be fine

Reply:
This `json_data ` is used in some parts of `artifactory.py`, eg: in the `rest_patch` method.

Reply:
Do we need to be consistent?

Reply:
@donhui you're 100% right! Consistency is more important.